frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx

```diff
@@ -125,6 +121,50 @@ export default function OntologyDagView({
     setHeight(newHeight);
   }, [screenDimensions, isFullScreen, inputWidth, inputHeight]);
 
+  const { data: markerGenePresence, isLoading: isLoadingMarkerGenePresence } =
+    useMarkerGenePresenceQuery();
+
+  const selectedTissue =
+    tissueName === TISSUE_AGNOSTIC ? ALL_TISSUES : tissueName;
+
+  const cellTypesWithMarkerGeneStats: {
```

Comment:
this memo converts the array of marker gene stats to an object keyed by cell types